src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/IllegalTypeCheck.java

```diff
@@ -144,7 +142,6 @@
     /** Creates new instance of the check. */
     public IllegalTypeCheck() {
         setIllegalClassNames(DEFAULT_ILLEGAL_TYPES);
-        setLegalAbstractClassNames(DEFAULT_LEGAL_ABSTRACT_NAMES);
```

Comment:
removed because we are replacing an empty array with an empty array. No way to confirm the change.